webserver/src/main/java/io/kestra/webserver/controllers/FlowController.java

```diff
@@ -445,8 +445,8 @@ public List<ValidateConstraintViolation> validateFlows(
         @Parameter(description = "A list of flows") @Body String flows
     ) {
         AtomicInteger index = new AtomicInteger(0);
-        return Stream
-            .of(flows.split("---"))
+        List<ValidateConstraintViolation> collect = Stream
```

Comment:
Directly returns the list

however, lgtm

Reply:
Ups, was for testing forgot it :+1: